app/views/hcb_codes/_wise_transfer.html.erb

```diff
@@ -1,6 +1,6 @@
-<% if @hcb_code.wise_transfer.wise_id.present? %>
+<% if @hcb_code.wise_transfer.wise_url %>
```

Comment:
nit: do we still want to call `.present?` here

Reply:
We do already in the implementation of `wise_url` so didn't think it necessary. Happy to add it if you think it's clearer.